ui/components/app/security-provider-banner-alert/blockaid-banner-alert/blockaid-banner-alert.js

```diff
@@ -12,10 +12,14 @@ import { I18nContext } from '../../../../contexts/i18n';
 import {
   BlockaidReason,
   BlockaidResultType,
+  FALSE_POSITIVE_REPORT_BASE_URL,
   SecurityProvider,
+  UTM_SOURCE,
 } from '../../../../../shared/constants/security-provider';
 import SecurityProviderBannerAlert from '../security-provider-banner-alert';
 
+const zlib = require('zlib');
```

Comment:
Hey, do we need to `require` zlib.

Reply:
Yea, this is from the blockaid reference PR.